lasagne/random.py

```diff
@@ -0,0 +1,26 @@
+"""
+A package-wide random number generator, used for weight initialization
+and seeding noise layers. This can be replaced by a
+numpy.random.RandomState object with a particular seed to facilitate
+reproducibility.
+"""
+
+import numpy as np
+
+
+_rng = np.random
+
+
+def get_rng():
+    """Get the package-level random number generator.
+    :return: a numpy.random or a numpy.random.RandomState object
+    """
```

Comment:
We're using numpydoc throughout the package, see [the numpydoc howto](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) for a detailed treatment or any file in Lasagne for examples. Here, I'd suggest:
```
Get the package-level random number generator for Lasagne.

Returns
-------
:class:`numpy.random.RandomState` instance
    The :class:`numpy.random.RandomState` instance passed to the most
    recent call of :func:`set_rng`, or ``np.random`` if :func:`set_rng` has
    never been called.
```